src/main/java/net/minecraftforge/gradle/user/UserBasePlugin.java

```diff
@@ -204,6 +220,34 @@ public void execute(TaskSourceCopy t)
             project.getTasks().getByName("build").dependsOn(TASK_SRC_JAR);
         }
 
+        // setup reobf tasks
+        @SuppressWarnings("unchecked")
+        NamedDomainObjectContainer<ReobfConfig> container = (NamedDomainObjectContainer<ReobfConfig>) project.getExtensions().getByName(EXT_REOBF);
+        for (ReobfConfig reobf : container)
+        {
+            Task jar = project.getTasks().getByName(reobf.getName());
+            if (!(jar instanceof Jar))
+            {
+                throw new GradleConfigurationException(reobf.getName() + "  is not a jar task. Can only reobf jars!");
+            }
+            String name = "reobf" + Character.toUpperCase(reobf.getName().charAt(0)) + reobf.getName().substring(1);
+            TaskSingleReobf task = maybeMakeTask(name, TaskSingleReobf.class);
+
+            task.setExceptorCfg(delayedFile(EXC_SRG));
+            task.setFieldCsv(delayedFile(CSV_FIELD));
+            task.setMethodCsv(delayedFile(CSV_METHOD));
+
+            task.setClasspath(reobf.getClasspath());
+            task.setJar(((Jar) jar).getArchivePath());
+            task.setPrimarySrg(delayedFile(reobf.getSrg()));
+
+            task.dependsOn(TASK_GENERATE_SRGS, jar);
+            task.mustRunAfter("test");
+
+            project.getTasks().getByName("build").dependsOn(task);
+            project.getTasks().getByName("assemble").dependsOn(task);
```

Comment:
`build` already depends on `assemble` so having both of these statements is redundant.